nautobot_circuit_maintenance/api/urls.py

```diff
@@ -8,6 +8,8 @@
 router.register("note", views.MaintenanceNoteTaskView)
 router.register("circuitimpact", views.MaintenanceCircuitImpactTaskView)
 router.register("notificationsource", views.NotificationSourceTaskView)
+router.register("parsednotification", views.ParsedNotificationTaskView)
```

Comment:
why do we need to add these?

Reply:
Some tests were failing, trying to access these URLs, so I added it here.